src/UI/templates/default/Button/button.less

```diff
@@ -2,4 +2,41 @@ a.btn-link.ilSubmitInactive, a.btn-link.ilSubmitInactive:hover {
 	background-color: inherit;
 	color: @text-color;
 	padding: 0;
-}
\ No newline at end of file
+}
+
+.btn-tag {
+	&.btn-tag-inactive {
+		cursor: default !important;
+	}
+	&.btn-tag-relevance-verylow {
+		background-color: desaturate(@btn-tag-bg, 80%);
+		color: contrast(desaturate(@btn-tag-bg, 80%));
```

Comment:
I can not answer that. I leave that to @catkrahl to decide. However, desaturate(@btn-tag-bg, 80%) and contrast(desaturate(@btn-tag-bg, 80%)) can not be set here. They must be moved to seperate less variables. See: https://github.com/ILIAS-eLearning/ILIAS/tree/trunk/templates#colors